### ISSUE TITLE
links were not made to things in lists?

### ISSUE THREAD

**Issue by kyook** [bioentity/Bioentity.link#32]
https://bioentity.link//#/publication/10.1534/genetics.119.302625

The article did not have any of the genes, alleles etc. within genotypes linked - due to <list> formatting?  

The listed objects were not viewable in the paper viewer.  Are the contents of <lists> </lists> tags set to be hidden? Can we unhide those and link them?

**Comment by nickstiffler** [comment]
Although the pub is locked, I am not sure what you are describing here. It sounds like something wasn't linked and also something is hidden.

**Comment by kyook** [comment]
@nickstiffler items in lists are hidden, but need to be seen and linked:
https://bioentity.link/#/publication/10.1534/g3.119.400639

**Comment by nickstiffler** [comment]
I updated texture to support lists. 
![list](https://user-images.githubusercontent.com/2396480/65457227-f6e34800-ddff-11e9-97a5-ddef1ed9c9c2.png)

**Comment by kyook** [comment]
Thank @nickstiffler oddly, i was able to link genes, but I wasn't able to link using a second keyword set.  That is I wasn't able to link the list section with wb_arp.  I linked the alleles in the list object by object.

**Comment by nickstiffler** [comment]
The indexer wasn't using lists. The genes were indexed elsewhere in the document, but the alleles only appear in the list. That should be fixed now.

**Comment by kyook** [comment]
not sure I understand this.   the genes in the doc and the list did get linked when I ran the linking script but the alleles in the list were not linked until I linked them manually.  By indexer are you referring to the side bar list of objects?

**Comment by nickstiffler** [comment]
Before it get linked, the document is parsed and a list is generated of words that could be linked. This list is then compared to the lexicon and the matches are sent to texture for the actual linking. If a word is missed initially (such as the words in the list), it won't be in the lexicon list that is sent to texture. In the case of the genes, they were found elsewhere in the article and made it into the lexicon subset that was sent to texture.
TL;DR: If something doesn't get linked, check the pub index on the bottom of the right side. If it isn't there, then it won't get linked.

**Comment by kyook** [comment]
ok, so in this case, the article was initially parsed, which missed parsing the terms in the list. 
So in order for the unique terms in the list to be parsed (and linkable), I would have had to load the article into the system again after you fixed it so that terms in lists are included?

**Comment by nickstiffler** [comment]
Under pub index, there is a "Clear" button. Use that to clear it and it will regenerate the index when linked.
